src/sentry/templates/sentry/toolbar/iframe.html

```diff
@@ -98,6 +98,28 @@
           return `${cookie}; domain=${domain}; path=/; max-age=31536000; SameSite=none; partitioned; secure`;
         }
 
+        const controlSiloPathPatterns = [
```

Comment:
no. it's in the sentry repo all over the place.

Reply:
Will this list need to change in the future as we add more control silo endpoints, or is this a fixed list based on what endpoints the toolbar uses?

Reply:
Right now it's the whole list, I was kind of expecting that it could be kept up to date as control endpoints are added. However only `/users/me` is needed currently.

What's likely to happen is this list won't be updated, and if a new url is needed by the toolbar it'll get discovered in as an error in prod. Ideally, the full list is here (or imported into here?) then that wouldn't happen. But idk... how many other places do we maintain that list of paths, and how often are new paths added?

Reply:
We might need to remove the regionUrl stuff though. It's looking like we can't set partitioned cookies for the region subdomain from an iframe loaded on the org-subdomain.

Reply:
> But idk... how many other places do we maintain that list of paths, and how often are new paths added?

There is a copy in webpack config, and one in a generated file that the rest of the frontend uses. New control silo endpoints are rare, and likely even more rare will be their usage in the toolbar.

Reply:
ok, i'm having a problem with regionUrl anyway which is more pressing. I think i will look at either: importing the generated file that the frontend uses, or just hard-coding the paths we need in a followup.

For now i'm going to switch back to all requests running through `organizationUrl` instead of `regionUrl`